openstack/python/tar/bin/openstack-describe-instances

```diff
@@ -32,7 +32,8 @@ class OpenStackDescribeInstances(DescribeInstancesCommand, OpenStackCommand):
                  2: 'Terminated',
                  3: 'Pending',
                  4: 'Unknown',
-                 5: 'Stopped'}
+                 5: 'Stopped',
+                 7: 'Error'}
```

Comment:
[libcloud.compute.base.NodeState](https://libcloud.readthedocs.org/en/latest/apidocs/libcloud.compute.html?highlight=nodestate#libcloud.compute.base.NodeState) this is an old reference.  
The one below is for v0.18, which we use on production now. http://libcloud.readthedocs.org/en/v0.18.0/apidocs/libcloud.compute.html?highlight=nodestate#libcloud.compute.base.NodeState
The implementation of the `libcloud.compute.base.NodeState` class was updated since then.  It now has `to/fromstring()` methods.
Due to lack of time I'm filling this map manually now.  I'll add an issue to do the mapping properly via that class.